simple/entities/README.md

```diff
@@ -0,0 +1,25 @@
+# Simple Entities Importer
+
+Outputs `observations.csv` for simple entities to be used by [RSI sqlite][sqlite].
+
+[sqlite]: https://github.com/datacommonsorg/mixer/tree/a768446c56095aa23add8c59cf6a0630a17a726b/internal/sqlite
+
+## Usage
```

Comment:
How are we dealing with unit, have a TODO maybe?

Reply:
Added TODO.

The original RSI doc talks about specifying them at the column or cell level. Another approach could be to allow separate columns for different properties. Let's decide on the approach and then add support in here.